src/Algebra/Construct/Flip/Op.agda

```diff
@@ -53,6 +53,14 @@ module _ (≈ : Rel A ℓ) (∙ : Op₂ A) where
   inverse : Inverse ≈ ε ⁻¹ ∙ → Inverse ≈ ε ⁻¹ (flip ∙)
   inverse inv = Prod.swap inv
 
+  zero : Zero ≈ ε ∙ → Zero ≈ ε (flip ∙)
+  zero zer = Prod.swap zer
+
+  module _ (+ : Op₂ A) where
+
+    distributes : (≈ DistributesOver ∙) + → (≈ DistributesOver (flip ∙)) +
```

Comment:
The other thing we could do to make this is a little nicer is to import `Algebra.Definitions` and `Algebra.Structures` locally in each anonymous module parameterised by the equality. Which would then allow us to avoid messing up the infix notation here?

Reply:
Ah... will now turn to this. 

And it seems that this applies throughout, even to the original module contents: so I've now named each previously anonymous module, and opened it locally in place with the appropriate arguments. is this better now? See most recent commit.